plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/SemgrepReader.java

```diff
@@ -19,11 +19,7 @@
 
 import org.json.JSONArray;
 import org.json.JSONObject;
-import org.owasp.benchmarkutils.score.BenchmarkScore;
-import org.owasp.benchmarkutils.score.CweNumber;
-import org.owasp.benchmarkutils.score.ResultFile;
-import org.owasp.benchmarkutils.score.TestCaseResult;
-import org.owasp.benchmarkutils.score.TestSuiteResults;
+import org.owasp.benchmarkutils.score.*;
```

Comment:
I prefer not to use .* on imports. Can you revert this here and in any other of the pull requests you have?

Reply:
sorry about that! fixed in this PR, will follow in the others ☺️